node/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "hangouts-chat-node-samples",
+  "version": "1.0.0",
+  "description": "Node samples for the Hangouts Chat API.",
+  "private": "true",
+  "keywords": [
+    "Hangouts Chat"
+  ],
+  "author": "Eric Koleda",
```

Comment:
Want to add "Eric Koleda, Barathwaja S"?

Reply:
I haven't added anything meaningful, so I'll leave my name off for now.